query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/fields/config/LdapDirectorySettingsField.java

```diff
@@ -150,83 +135,66 @@ public String memberAttributeReferencedInGroup() {
         return memberAttributeReferencedInGroup.getValue();
     }
 
-    public LdapSettingsField useDefaultAuthentication() {
-        useCase.isRequired(true);
+    public LdapDirectorySettingsField useDefaultUser() {
         baseUserDn.isRequired(true);
-        baseGroupDn.isRequired(true);
         usernameAttribute.isRequired(true);
+        return this;
+    }
+
+    public LdapDirectorySettingsField useDefaultAuthentication() {
```

Comment:
We should rename this method to indicate the fields are being set to required. something like `useDefaultRequiredForAuthentication`